MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java

```diff
@@ -161,10 +169,10 @@ public void onAdsManagerLoaded(AdsManagerLoadedEvent adsManagerLoadedEvent) {
 
                     // Set up the ad events that we want to use
                     AdsManager adsManager = adsManagerLoadedEvent.getAdsManager();
-                    AdsImaSDKListener imaListener = new AdsImaSDKListener(baseExoPlayer);
+
```

Comment:
The above change means that we need to instantiate the `adsImaSdkListener` here, only within this call, where we have it guarded in a `try` checking for all the classes we need.